### DISCUSSION TITLE
Support for RWA token and future private equity tokens

### DISCUSSION THREAD

**Discussion by Yohann76** [RealToken-Community/dashboard-v2#54] [General]
Hello, has the support for the RWA token in the dashboard already been discussed? In what form?

**Comment by daodevinc** [comment]
@jycssu-com @NandyBa do you already planned this feature ?

**Reply by Yohann76** [reply]
Apparently, it's already planned; a pull request from @NandyBa  is currently under review on this matter

**Reply by daodevinc** [reply]
Yes on https://github.com/RealToken-Community/dashboard-v2/pull/61

**Reply by NandyBa** [reply]
PR will be ready on the coming days

**Reply by NandyBa** [reply]
You can now provide review #61
